planner/core/planbuilder.go

```diff
@@ -652,6 +652,21 @@ func (b *PlanBuilder) detectSelectWindow(sel *ast.SelectStmt) bool {
 	return false
 }
 
+func checkReadEngineHasTiKV(ctx sessionctx.Context) bool {
+	cfgIsolationEngines := set.StringSet{}
+	for _, engine := range config.GetGlobalConfig().IsolationRead.Engines {
```

Comment:
We don't need to check config. Since the session variable is inited by it.

Reply:
I see the function `filterPathByIsolationRead` also check it.

Reply:
The new code doesn't check.

Reply:
Addressed.